src/pages/Home.jsx

```diff
@@ -1,23 +1,28 @@
 import MotivationalQuotes from "../components/MotivationalQuotes";
 import JobList from "../components/JobList";
 import "../index.css";
+import { NavLink } from "react-router-dom";
 
 function Home() {
+  const username = localStorage.getItem("userName");
   return (
     <div className=" bg-softPink flex flex-col items-left justify-start p-5 gap-5">
-      <div className="flex flex-row">
-        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl  font-lora border border-none rounded-3xl p-8 w-1/2 text-left hover:border-2 hover:border-solid hover:border-blue-500">
-          {" "}
-          Hi [user!]
-        </div>
-        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl font-lora border border-none rounded-3xl p-8 w-1/4 text-left hover:border-2 hover:border-solid hover:border-blue-500 hover:underline hover:decoration-blue-500">
-          {" "}
-          Job Tracking
-        </div>
-        <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl font-lora border border-none rounded-3xl p-8 w-1/4 text-left hover:border-2 hover:border-solid hover:border-blue-500 hover:underline hover:decoration-blue-500">
-          {" "}
-          Resources
-        </div>
+      <div className="flex flex-row gap-5">
+        <NavLink to={"/"} className="w-1/2">
+          <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl  font-lora border border-none rounded-3xl p-8 text-left hover:border-2 hover:border-solid hover:border-blue-500">
+            Hi {username}
+          </div>
+        </NavLink>
```

Comment:
```suggestion
          <div className="bg-linear-to-r from-gray-50 to-white shadow-md ml-8 text-3xl  font-lora border border-none rounded-3xl p-8 text-left hover:border-2 hover:border-solid hover:border-blue-500">
            Hi {username}
          </div>
```

Remove the `NavLink` for this `div` -- it is just a label for the page.

Reply:
I will make the changes.